heron/instance/src/java/com/twitter/heron/instance/HeronInstance.java

```diff
@@ -237,8 +237,7 @@ private void handleException(Thread thread, Throwable exception) {
       if (exception instanceof Error) {
         LOG.log(Level.SEVERE,
             "Error caught in thread: " + thread.getName()
-                + " with thread id: " + thread.getId() + ". Process halting...",
```

Comment:
If the concern is only OOM errors I say we check for that and log a message that it's an OOM without logging the error. If it's anything else, we should log it.